common/autoware_auto_perception_rviz_plugin/src/object_detection/object_polygon_detail.cpp

```diff
@@ -392,6 +430,49 @@ void calc_bounding_box_line_list(
   points.push_back(point);
 }
 
+void calc_2d_bounding_box_line_list(
```

Comment:
```suggestion
void calc_2d_bounding_box_bottom_line_list(
```
maybe better to use the word (`bottom`)

Reply:
changed